roads.mss

```diff
@@ -393,6 +394,25 @@ as well. */
 }
 
 /* ================================================================== */
+/* FERRIES
+/* ================================================================== */
+
+#ferry {
```

Comment:
If `route=ferry` is a subset of `#ferry`, so we should add the filter on the main selector (`#ferry[route='ferry']`) otherwise we can have non ferry routes displayed with default style. Other option, that may the best, is to filter out in the SQL itself so we only have the routes we are interested in.